packages/react/use-overlay/src/useOverlay.en.md

```diff
@@ -72,7 +72,4 @@ const openFooConfirmDialog = () => {
 };
 
 await openFooConfirmDialog();
-
```

Comment:
I removed the log stating 'dialog closed' because I was concerned that people might misunderstand that a log appears in the console window when the Overlay is closed. What do you think about expressing this through comments or other methods?

Reply:
If you can express how developer can perform the actionafter the overlay is closed, you can do anything 

I don't think the current console will be too confusing

Reply:
I think there's no other good way. Thank you!